src/main/java/com/exasol/releaserobot/repository/maven/MavenPlugin.java

```diff
@@ -0,0 +1,91 @@
+package com.exasol.releaserobot.repository.maven;
+
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+
+/**
+ * This class represents a Maven plugin.
+ */
+public class MavenPlugin {
+    private final String artifactId;
+    private final Xpp3Dom configuration;
```

Comment:
I think for now it's ok, but you should consider to create an interface for the `configuration`, if at some point in the future you use another library for it.